CHANGELOG.md

```diff
@@ -2,6 +2,15 @@
 
 All notable changes to this project will be documented in this file. For future plans, see our [Roadmap](https://github.com/precice/precice/wiki/Roadmap).
 
+## 2.1.1
+
+- Fix compilation error emitted by intel compilers when compiling tests.
+- Fix error message for unique acceleration subtags.
+- Fix parsing error on systems without locales installed. This fixes issues when running preCICE in minimal docker containers.
+- Fix preCICE erroneously expects cyclic communicator for IMVJ with restart.
+- Fix system-dependent compilation error when compiling tests.
+- Update the PKGBUILD for the Arch Linux User Repository to be compatible with preCICE v2.
+
```

Comment:
Should we mention our migration from Travis to github actions here? Not really relevant for the users, but an important step.

Reply:
I wouldn't mention it here, as a normal user never even gets in touch with the CI. It would be a good topic for a blogpost though.